app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -157,11 +157,17 @@ def open_claims
   def all_rated_disabilities
     settings = Settings.lighthouse.veteran_verification.form526
     icn = UserAccount.where(id: user_account_id).first&.icn
-    service = ApiProviderFactory.rated_disabilities_service_provider(
-      { auth_headers:, icn: }
+    api_provider = ApiProviderFactory.call(
+      type: ApiProviderFactory::FACTORIES[:rated_disabilities],
+      provider: nil,
+      options: { auth_headers:, icn: },
+      # Flipper id is needed to check if the feature toggle works for this user
+      current_user: OpenStruct.new({ flipper_id: user_account_id }),
```

Comment:
Does this change what we would need to use in the "Logged in Users" section of the Flipper UI? Currently we are entering their emails, would we need to enter this `user_account_id` into the box instead?

Reply:
yes, probably.
this is how the `flipper_id` works from the `User.rb` model line 416-ish
```
def flipper_id
    email&.downcase || account_uuid
end
```
since we do not have access to the user's email in the background job context, we will probably need to use the account_uuid instead of the email for rated disabilities.